google-cloud-flatcar-image-upload/upload_image.sh

```diff
@@ -15,7 +15,7 @@ Usage: $0 [OPTION...]
 
  Required arguments:
   -b, --bucket-name Name of GCP bucket for storing images.
-  -p, --project-id  ID of the project for creating bucket.
+  -p, --project-id  ID of the project for creating bucket. Not required if --skip-auth is used.
```

Comment:
I wonder if this is still "Required" argument, if it's "Optional" in some cases :smile: